.circleci/config.yml

```diff
@@ -6,16 +6,13 @@ defaults: &defaults
 
   working_directory: ~/app
 
-
-
-
-
 jobs:
   commitlint:
     docker:
-      - image: williamlauze/circleci-commitlint:latest
+      - image: secretbase/circleci-commitlint:1.0.1
```

Comment:
I used the fixed version here to prevent breaking changes, If I mess up anything when updating the code.